src/Utility/Security.php

```diff
@@ -66,6 +66,15 @@ public static function hash($string, $type = null, $salt = false)
         }
         $type = strtolower($type);
 
+        $hashAlgos = hash_algos();
+        if (!in_array($type, $hashAlgos)) {
+            throw new RuntimeException(sprintf(
+                'The hash type `%s` was not found. Available algorithms are: %s',
```

Comment:
instead of type:
`hash algorithm`
`hash algo`
The function variable could also be renamed to `$algo`.